packages/mask/src/plugins/ITO/SNSAdaptor/ClaimAllDialog.tsx

```diff
@@ -227,9 +227,15 @@ export function ClaimAllDialog(props: ClaimAllDialogProps) {
     } = useSpaceStationCampaignInfo(account, Flags.nft_airdrop_enabled)
 
     const [chainId, setChainId] = useState(
-        [ChainId.Mainnet, ChainId.BSC, ChainId.Matic, ChainId.Arbitrum, ChainId.xDai, ChainId.Fantom].includes(
-            currentChainId,
-        )
+        [
+            ChainId.Mainnet,
+            ChainId.BSC,
+            ChainId.Matic,
+            ChainId.Arbitrum,
+            ChainId.xDai,
+            ChainId.Fantom,
+            ChainId.Avalanche,
+        ].includes(currentChainId)
```

Comment:
extract the array as constants.

e.g: `ALLOWED_CHAIN_ID_LIST`

Reply:
done